### PR TITLE
add support for running tests using mvn. removed dup dep from pom.xml

### PR DESCRIPTION
**Pull request by bitcoder**
This PR adds the ability to run tests using Maven right from the command-line.
I followed the convention where Test classes end up in "Test" so I've renamed them.
Also removed a duplicated/conflicting dependency on rest assured.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,4 +60,7 @@ atlassian-ide-plugin.xml
 com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
-fabric.properties
\ No newline at end of file
+fabric.propertiesa
```
Comment (basdijkstra) [inline .gitignore:63]: What is this `.propertiesa` entry? I appreciate you adding the VS Code settings to be excluded, but I don't understand this bit.

File: pom.xml [hunk 2]
```diff
@@ -11,7 +11,6 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <restassured.version>4.3.1</restassured.version>
-        <restassured.version>4.2.0</restassured.version>
```
Comment (basdijkstra) [inline pom.xml:14]: Thanks, I forgot to remove that!

File: src/test/java/answers/RestAssuredAnswers1Test.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import io.restassured.specification.*;
 import org.junit.*;
 
-public class RestAssuredAnswers1 {
+public class RestAssuredAnswers1Test {
```
Comment (basdijkstra) [inline src/test/java/answers/RestAssuredAnswers1Test.java:12]: That makes sense... I removed the `Test` suffixes in this latest release, but I knew there was a reason for me adding them initially. Now I remember :)

Obviously, the same goes for all the other name changes, highly appreciated.

**Review by basdijkstra — COMMENTED** [review]
Hey @bitcoder, could you please have a look at the first comment? If that's sorted out, I'm happy to merge this PR.

If it was a typo (as I suspect), please use a fixup commit to prevent the commit history from getting too cluttered:

https://jordanelver.co.uk/blog/2020/06/04/fixing-commits-with-git-commit-fixup-and-git-rebase-autosquash/

File: .gitignore (hunk 1 above)
Comment (bitcoder) [reply .gitignore:63]: it was a typo for sure. fixing it..

**Review by basdijkstra — CHANGES_REQUESTED** [review]
Sorry to bother you again, but could you please rebase with autosquash against master? To do that, go to your branch, then

`git rebase -i --autosquash origin/master`

and then

`git push --force-with-lease`

This squashes the fixup commit 'into' the original commit, keeping the commit history clean. Thanks :)

**Comment by bitcoder** [comment]
hi @basdijkstra , can you please check if now is ok?

**Comment by basdijkstra** [comment]
Perfect, thank you for the PR, @bitcoder. I really appreciate it. Merging.

**Comment by bitcoder** [comment]
you're welcome :)

**Pull request merged**